src/coprocessor/dag/batch_handler.rs

```diff
@@ -40,19 +41,10 @@ pub struct BatchDAGHandler {
 
     config: Arc<EvalConfig>,
 
-    /// Accumulated statistics.
-    // TODO: Currently we return statistics only once, so these statistics are accumulated only
-    // once. However in future when we introduce reenterable DAG processor, these statistics may
-    // be accumulated and returned several times during the life time of the request. At that time
-    // we may remove this field.
-    statistics: BatchExecuteStatistics,
-
-    /// Traditional metric interface.
-    // TODO: Deprecate it in Coprocessor DAG v2.
-    metrics: ExecutorMetrics,
```

Comment:
Do we really don't need the metrics? (also confused in https://github.com/tikv/tikv/pull/5121/files#diff-2fe84582387a14020db82f4371d0e77eL50 because  I cannot found the alternative)

Reply:
Sorry, I have found the alternative of this deleted part.

Reply:
Scan counter is never used in ansible dashboard. So related metrics are also removed.